### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by CarlosAmorimProjects**
Here you go. Can you see all files ?

### CODE REVIEW

File: src/pages/text/text.js [hunk 1]
```diff
@@ -95,6 +95,22 @@ export default function Text () {
...
+          <option value='es-es'>Spanish (Spain)</option> 
+          <option value='es-us'>Spanish (United States)</option>
+          <option value= 'sw'>Swahili</option>
+          <option value='sv'>Swedish</option>
+          <option value='ta'>Tamil</option>
+          <option value='th'>Thai</option>
+          <option value='tr'>Turkish</option>
+          <option value='uk'>Ukrainian</option>
+          <option value='vi'>Vietnamese</option>
+          <option value='cy'>Welsh</option>
```
Comment (mrf345) [inline src/pages/text/text.js]: What about having the `options` stored in the state as an object or array of objects. What if we wanted to add or remove  support for languages dynamically 🤔

File: utils.py [hunk 2]
```diff
@@ -2,16 +2,22 @@
...
-       files = glob.glob(os.path.join("/build/static/audiofiles/", '*.mp3'))
+       files = glob.glob(os.path.join("build/static/audiofiles/", '*.mp3'))
        if len(files)>limit:
            for file in files:  
              os.remove(file)
+
+def remove_pontuation (text):
+        text_clean = re.sub('\!|\?', '', text)
+        return text_clean
+
```
Comment (mrf345) [inline utils.py]: Last line should be empty of white spaces.

File: src/pages/instructions/instru.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ export default function Instru () {
             </h1>
           </div>
         <div className="instructions">
-          <h3>1. Enter text on screen <span aria-label="a rocket blasting off" role="img">💻</span> </h3>
+          <h3>1. Enter text on screen (Remove any extra punctuation marks) <span aria-label="a rocket blasting off" role="img">💻</span> </h3>
```
Comment (mrf345) [inline src/pages/instructions/instru.js]: Nice one, I like how you wrapped the emojis here ✨

**Review by mrf345 — APPROVED** [review]
Awesome work. Just small obsessive nitpicks 🚀

File: src/pages/text/text.js (hunk 1 above)
Comment (CarlosAmorimProjects) [reply src/pages/text/text.js]: It's a great idea, i will start working on it !

**Comment by CarlosAmorimProjects** [comment]
Did the select menu the way you sugested, i will try to put it in separate file so text.js is easier to read.

**Pull request closed without merging**